contracts/contracts/vault/VaultCore.sol

```diff
@@ -300,25 +300,26 @@ contract VaultCore is VaultStorage {
         for (uint256 i = 0; i < allAssets.length; i++) {
             IERC20 asset = IERC20(allAssets[i]);
             uint256 assetBalance = asset.balanceOf(address(this));
-            // No balance, nothing to do here
-            if (assetBalance == 0) continue;
```

Comment:
Ugh. I'm assuming this change is for slither. 

Does it get happier if we just do < 1 ? I really like the return/continue early style rather than nesting code in ifs.